DrivewayDentDeletion/Operators/prereqs.sh

```diff
@@ -76,90 +99,78 @@ oc adm policy add-scc-to-group privileged system:serviceaccounts:${test_namespac
+  namespace: $image_project
+  name: postgres-credential
+type: Opaque
 stringData:
-  username: ${ER_USERNAME}
-  password: ${ER_PASSWORD}
+  username: $DB_USER
+data:
+  password: ${PASSWORD_ENCODED}
 EOF
 
-echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
-
-echo "Waiting for postgres to be ready"
-oc wait -n postgres --for=condition=available deploymentconfig --timeout=20m postgresql
+  echo -e "INFO: Configuring postgres in the '$image_project' namespace with the user '$DB_USER' and database name '$DB_NAME' and suffix '$SUFFIX'\n"
+  if ! ${CURRENT_DIR}/../../products/bash/configure-postgres.sh -n ${POSTGRES_NAMESPACE} -u $DB_USER -d $DB_NAME -p $DB_PASS; then
+    echo -e "\n$cross ERROR: Failed to configure postgres in the '$image_project' namespace with the user '$DB_USER' and database name '$DB_NAME' amd suffix '$SUFFIX'"
+    exit 1
+  else
+    echo -e "\n$tick INFO: Successfuly configured postgres in the '$image_project' namespace with the user '$DB_USER' and database name '$DB_NAME' and suffix '$SUFFIX'\n"
+  fi  #${CURRENT_DIR}/../../products/bash/configure-postgres.sh -n ${image_project} -u $DB_USER -d $DB_NAME -p $DB_PASS
+
+  echo "INFO: Creating the table 'QUOTES' in the database '$DB_NAME' with the username '$DB_USER' in the '$image_project' namespace"
+  if ! oc exec -n $POSTGRES_NAMESPACE -it $DB_POD \
+      -- psql -U $DB_USER -d $DB_NAME -c \
+    'CREATE TABLE IF NOT EXISTS QUOTES (
+      QuoteID SERIAL PRIMARY KEY NOT NULL,
+      Name VARCHAR(100),
+      EMail VARCHAR(100),
+      Address VARCHAR(100),
+      USState VARCHAR(100),
+      LicensePlate VARCHAR(100),
+      ACMECost INTEGER,
+      ACMEDate DATE,
+      BernieCost INTEGER,
+      BernieDate DATE,
+      ChrisCost INTEGER,
+      ChrisDate DATE);'; then
+    echo -e "\n$cross ERROR: Failed to create the table 'QUOTES' in the database '$DB_NAME' with the username '$DB_USER' in the namespace '$image_project'"
+    exit 1
+  else
+    echo -e "\n$tick INFO: Created the table 'QUOTES' in the database '$DB_NAME' with the username '$DB_USER' in the namespace '$namespace'"
+  fi
 
-echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+  echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
 
-declare -a image_projects=("${dev_namespace}" "${test_namespace}")
-declare -a suffix=("ddd")
+  echo -e "INFO: Creating ace postgres configuration and policy in the namespace '$image_project' with the user '$DB_USER' and database name '$DB_NAME' and suffix '$SUFFIX'"
+  if ! ${CURRENT_DIR}/../../products/bash/create-ace-config.sh -n ${image_project} -g $POSTGRES_NAMESPACE -u $DB_USER -d $DB_NAME -p $DB_PASS; then
```

Comment:
I think this script is not really general purposes. It's creating an ACE `Configuration` that is specific to DDD, referring to the DDD MQ and the DDD postgres database. Maybe this should be in the DDD dir. And also use a name for the `Configuration` that includes `ddd` (currently it uses `ace-policyproject`).